lib/pbench/test/unit/server/test_requests.py

```diff
@@ -28,36 +29,32 @@ def test_host_info(client, pytestconfig, caplog):
 
 class TestElasticsearch:
```

Comment:
We should consider breaking these test classes off into separate files associated with each of the new resource classes.

(I'm not really pointing a finger at you, since I was the first to break off the new `QueryControllers` and I was lazy about this... but we should still think about doing it!)

Reply:
yep I was thinking about that, will just go ahead then

Reply:
Good start; we can unbundle the rest later. (And I suspect we'll want to generalize the `query_helper` fixture to mock behavior on the GraphQL and Elasticsearch pass-through APIs as well.)